iothub/device/devdoc/amqpTransportExceptions.md

```diff
@@ -1,35 +1,57 @@
-Below is the behavior of the SDK on receiving an exception from the AMQP library being used underneath. If the exception is marked as retryable, the SDK will implement the default retry-policy and attempt to reconnect. For exceptions not marked as retryable, it is advised to inspect the exception details and perform the necessary action as indicated below.
+Below is the behavior of the SDK on receiving an exception over Amqp transport protocol. If the exception is marked as retryable, the exception will get translated to an `AmqpIoTResourceException` internally, and the SDK will implement the default retry-policy and attempt to reconnect. For exceptions not marked as retryable, it is advised to inspect the exception details and perform the necessary action as indicated below.
```

Comment:
Do we need to specify the implementation details on what exception it gets converted to here? This might get out of sync if we change the name of the exception. Might be better to keep it generic.

Reply:
When our retry policy expires/ an operation is carried out when the client is disconnected, then the inner exception has this `AmqpIoTResourceException` referenced. In the past customers have asked about this internal exception, so I would like to have this mentioned here.